MetaMorpheus/TaskLayer/SearchTask/SearchTask.cs

```diff
@@ -166,22 +166,22 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
             // write prose settings
             ProseCreatedWhileRunning.Append("The following search settings were used: ");
-            ProseCreatedWhileRunning.Append("protease = " + CommonParameters.DigestionParams.DigestionAgent + "; ");
+            ProseCreatedWhileRunning.Append($"{(GlobalVariables.AnalyteType == AnalyteType.Oligo ? "Rnase" : "protease")} = " + CommonParameters.DigestionParams.DigestionAgent + "; ");
```

Comment:
You use this exact ternary a couple times. Maybe just add a property to GlobalVariables or a method to AnalyteType like GetDigestionAgentLabel()?